ims/common/shell.py

```diff
@@ -0,0 +1,20 @@
+# This is a wrapper for subprocess that must be used by all shell calls
+# The sudo in this doesnt take password as we wont be needing it.
+# shell=True must never be used
```

Comment:
The reason I put it so that other new developers do not use shell=True by mistake